tools/coverage/README.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+Code coverage tool has two major features.
+
+1. As a pre-submit tool, it runs code coverage on every single commit to Github and reports coverage changes back to the PR as a comment by a robot account. If made an required job on repo, it can be used to block a PR from merging if coverage falls below threshold
+1. As a periodical running job, It outputs a junit xml that can be read from other tools like [testgrid](http://testgrid.knative.dev/serving#coverage) to get overall coverage metrics
+
+## Design
+
+See [design.md](https://github.com/knative/test-infra/tree/master/tools/coverage/design.md)
+
+## Build & Release
```

Comment:
Replace "dev version" with "staging version" and explain what it is and what's the intended use.

Reply:
You didn't explain what's the intended use.